site/docs/master/httpproxy.md

```diff
@@ -673,14 +673,15 @@ In this example, requests to `timeout.bar.com/` will have a response timeout pol
 This refers to the time that spans between the point at which complete client request has been processed by the proxy, and when the response from the server has been completely processed.
 
 - `timeoutPolicy.response` This field can be any positive time period or "infinity".
-The time period of **0s** will also be treated as infinity.
 This timeout covers the time from the *end of the client request* to the *end of the upstream response*.
 By default, Envoy has a 15 second value for this timeout.
 More information can be found in [Envoy's documentation][4].
+Note that a value of **0s** will be treated as if the field were not set, i.e. by using Envoy's default behavior.
```

Comment:
Can you please also update the API documentation in `apis/projectcontour/v1/httpproxy.go`, which is the authoritative reference for the HTTPProxy spec?

Reply:
added the same note to the API docs